Makefile

```diff
@@ -22,6 +22,8 @@ COMPONENTS=cloud \
 .EXPORT_ALL_VARIABLES:
 OUT_DIR ?= _output/local
```

Comment:
Set a default value for BUILD_WITH_CONTAINER here? Let's build in container by default.

Reply:
yes, but seems that we need more fix to adjust it, will open another PR to set build in container default
https://github.com/kubeedge/kubeedge/runs/5844143228?check_suite_focus=true

Reply:
Ok, if there're some actions build in container originally, we can disable `BUILD_WITH_CONTAINER`.